.travis.yml

```diff
@@ -1,17 +1,14 @@
 language: ruby
+sudo: false
+cache: bundler
+bundler_args: '--without production development'
 
 rvm:
   - 2.2.2
 
-env: DATABASE_URL=postgres://postgres@localhost/timeoverflow_test
```

Comment:
Well, production goes with DATABASE_URL, so it makes sense that CI goes with the same.